bm-persona/Libraries/LibraryDetailViewController.swift

```diff
@@ -0,0 +1,362 @@
+        card.leftAnchor.constraint(equalTo: scrollView.layoutMarginsGuide.leftAnchor).isActive = true
+        card.rightAnchor.constraint(equalTo: scrollView.layoutMarginsGuide.rightAnchor).isActive = true
+        card.heightAnchor.constraint(equalToConstant: 200).isActive = true
+        
+        let nameLabel = UILabel()
+        nameLabel.font = Font.semibold(20)
+        nameLabel.text = library!.searchName
+        card.addSubview(nameLabel)
+        nameLabel.lineBreakMode = .byWordWrapping
+        nameLabel.numberOfLines = 0
+        nameLabel.translatesAutoresizingMaskIntoConstraints = false
+        nameLabel.topAnchor.constraint(equalTo: card.layoutMarginsGuide.topAnchor).isActive = true
+        nameLabel.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
+        nameLabel.widthAnchor.constraint(equalTo: card.layoutMarginsGuide.widthAnchor, multiplier: 0.5).isActive = true
+        nameLabel.heightAnchor.constraint(equalTo: card.layoutMarginsGuide.heightAnchor, multiplier: 0.5).isActive = true
+        nameLabel.adjustsFontSizeToFitWidth = true
+        
+        let addressIcon = UIImageView()
+        addressIcon.image = #imageLiteral(resourceName: "Walk")
+        card.addSubview(addressIcon)
+        addressIcon.translatesAutoresizingMaskIntoConstraints = false
+        addressIcon.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
+        addressIcon.topAnchor.constraint(equalTo: nameLabel.layoutMarginsGuide.bottomAnchor, constant: 20).isActive = true
+        addressIcon.widthAnchor.constraint(equalTo: card.layoutMarginsGuide.widthAnchor, multiplier: 0.06).isActive = true
+        addressIcon.contentMode = .scaleAspectFit
+        
+        
+        let addressLabel = UILabel()
+        if let longAddress = library!.campusLocation {
+            if let ind = longAddress.range(of: "Berkeley")?.upperBound {
+                let newAdress = longAddress[..<ind]
+                addressLabel.text = String(newAdress)
+            } else {
+                addressLabel.text = longAddress
+            }
+        }
+        card.addSubview(addressLabel)
+        addressLabel.font = Font.thin(12)
+        addressLabel.numberOfLines = 2
+        addressLabel.textColor = UIColor.black
+        addressLabel.translatesAutoresizingMaskIntoConstraints = false
+        addressLabel.topAnchor.constraint(equalTo: nameLabel.layoutMarginsGuide.bottomAnchor, constant: 20).isActive = true
+        addressLabel.leftAnchor.constraint(equalTo: addressIcon.layoutMarginsGuide.rightAnchor, constant: 8).isActive = true
+        //addressLabel.widthAnchor.constraint(equalTo: card.layoutMarginsGuide.widthAnchor, multiplier: 0.5).isActive = true
+        addressLabel.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor, constant: -160).isActive = true
+        addressLabel.adjustsFontSizeToFitWidth = true
+        
+        
+        let phoneLabel = UILabel()
```

Comment:
I think the ordering of the elements needs to be updated to match the design.

See:
![image](https://user-images.githubusercontent.com/7193442/78409959-f2be9100-75bf-11ea-89a4-3cf38c1298fc.png)